integration_tests/src/main/python/json_test.py

```diff
@@ -81,6 +83,35 @@
 _string_schema = StructType([
     StructField('a', StringType())])
 
+json_supported_date_formats = [
+    None, # represents not specifying a format (which is different from explicitly specifying the default format in some Spark versions)
+    'yyyy-MM-dd', 'yyyy/MM/dd',
+    'yyyy-MM', 'yyyy/MM',
+    'MM-yyyy', 'MM/yyyy',
+    'MM-dd-yyyy', 'MM/dd/yyyy',
+    'dd-MM-yyyy', 'dd/MM/yyyy']
+
+json_supported_ts_parts = [
```

Comment:
Found that this is already added (differently): https://github.com/NVIDIA/spark-rapids/blob/328a51495fbc2181c0445919307a9e65a24afbff/integration_tests/src/main/python/json_test.py#L177

So should we modify that too?